src/compose.yaml

```diff
@@ -0,0 +1,50 @@
+services:
+  frontend:
+    container_name: frontend
+    user: "${USER_ID}:${GROUP_ID}" # prevent root files in host system
+    build:
+      context: ./deskstar-frontend
+    ports:
+      - 3000:3000
+    volumes:
+      - ./deskstar-frontend:/app/code
```

Comment:
Maybe we should cache the volume mount for better performance:
https://stackoverflow.com/questions/43844639/how-do-i-add-cached-or-delegated-into-a-docker-compose-yml-volumes-list

Reply:
Good point. I applied your changes.